nipype/interfaces/base/support.py

```diff
@@ -168,10 +168,8 @@ def _get_bunch_hash(self):
         sorted_dict = to_str(sorted(dict_nofilename.items()))
         return dict_withhash, md5(sorted_dict.encode()).hexdigest()
 
-    def __pretty__(self, p, cycle):
-        """Support for the pretty module
-
-        pretty is included in ipython.externals for ipython > 0.10"""
+    def _repr_pretty_(self, p, cycle):
```

Comment:
I'm okay with this, but we should verify that we can pretty print Bunches in IPython with this change. I don't know if the API changed beyond what method it looks for in your objects.

if it doesn't work quickly, I'd say remove the function altogether and create an issue to fix and re-add, rather than waste time on this today.

Reply:
It looks like what it does is prevents an infinite loop during formating. Could you test with a bunch that contains itself?

Reply:
yes, you're right, for a bunch that contains itself it makes a difference! But only with the new name `_repr_pretty_`. 
And you don't actually have to import pretty in ipython.